e2e/policy/policy_test.go

```diff
@@ -161,10 +162,13 @@ func TestPolicy(t *testing.T) {
 
 		// wait for the init container of the openssl-frontend pod to enter the running state
 		require.NoError(c.WaitFor(ctx, kubeclient.InitContainersRunning, kubeclient.Deployment{}, ct.Namespace, opensslFrontend))
-		newFailures := getFailures(ctx, t, ct)
-		t.Log("New failures:", newFailures)
-		// errors should happen
-		require.Greater(newFailures, initialFailures)
+
+		// The container may be running, but it's hard to tell whether it already had a chance to
+		// connect to the Coordinator. Thus, retry for some time until an attestation failure happens.
+		require.EventuallyWithT(func(t *assert.CollectT) {
+			newFailures := getFailures(ctx, t, ct)
+			require.Greater(newFailures, initialFailures, "pod not covered by manifest should cause attestation failure")
+		}, 1*time.Minute, 100*time.Millisecond)
```

Comment:
```suggestion
		}, 1*time.Minute, time.Second)
```
Not sure, might be enough to try this every sec?